### ISSUE TITLE
No error reported for SyntaxError: named arguments must follow bare *

### ISSUE THREAD

**Issue by NCPlayz** [microsoft/pyright#3855]
**Describe the bug**
(see code)

**Expected behavior**
Pyright should error saying that `named arguments must follow bare *`

**Screenshots or Code**
```python
def foo(bar: int, *, **baz: str) -> None:
    ...
```

**VS Code extension or command-line**
VS Code: pylance `v2022.8.30`
Command-line: pyright `v1.1.267`

**Comment by erictraut** [comment]
This will be included in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.268, which I just published. It will also be included in a future version of pylance.
